src/docs/components/styling.md

```diff
@@ -162,6 +162,45 @@ shadow-card::part(heading) {
 
 This allows for greater flexibility in styling as any styles can now be added to this element.
 
+### Exportparts
+
+If you have a Stencil component nested within another component, any `part` specified on elements of the child component will not be exposed through the parent component. In order to expose the `part`s of the child component, you need to use the `exportparts` attribute. Consider this `OuterComponent` which contains the `InnerComponent`.
+
+```tsx
+export class OuterComponent {
+  render() {
+    return (
+      <Host>
+        <h1>Outer Component</h1>
+        <inner-component exportparts="inner-text" />
+      </Host>
+    );
+  }
+}
+
+export class InnerComponent {
+  render() {
+    return (
+      <Host>
+        <h1 part="inner-text">Inner Component</h1>
+      </Host>
+    );
+  }
+}
+```
+
+By specifying "inner-text" as the value of the `exportparts` attribute, elements of the `InnerComponent` with a `part` of "inner-text" can now be sty;ed in the light DOM. Even though the `InnerComponent` is not used directly, we can style its parts through the `OuterComponent`.
```

Comment:
Not necessary ATM, one thing I want us to start thinking about is consistency when referring to components (consistency throughout the doc site that is).  At some point in the near future, I want to decide if we should refer to components by their tag (`outer-component`) vs their class (`OuterComponent`). We can always deviate where necessary to get a particular point across - this is something that's cropped up on my end while working through the props docs

Reply:
yeah I was thinking about this as well as I was writing it. Let's circle back to this